pkg/controllers/resources/csidrivers/syncer.go

```diff
@@ -47,12 +50,19 @@ func (s *csidriverSyncer) Syncer() syncertypes.Sync[client.Object] {
 
 func (s *csidriverSyncer) SyncToVirtual(ctx *synccontext.SyncContext, event *synccontext.SyncToVirtualEvent[*storagev1.CSIDriver]) (ctrl.Result, error) {
 	vObj := translate.CopyObjectWithName(event.Host, types.NamespacedName{Name: event.Host.Name, Namespace: event.Host.Namespace}, false)
+
+	// Apply pro patches
+	err := pro.ApplyPatchesVirtualObject(ctx, event.Host, vObj, event.Host, ctx.Config.Sync.FromHost.CSIDrivers.Patches)
```

Comment:
This needs to be `pro.ApplyPatchesVirtualObject(ctx, nil, vObj, event.Host, ctx.Config.Sync.FromHost.CSIDrivers.Patches)`